dle/dle/settings.py

```diff
@@ -35,14 +35,15 @@
 # Hosts and CIDR (AWS subnets)
 try:
     ALLOWED_HOSTS = [
+        "*",
         "druglabelexplorer.org",
         "www.druglabelexplorer.org",
         "127.0.0.1",
         "localhost",
         "testserver",
     ] + env.list("ALLOWED_HOSTS")
 except ImproperlyConfigured:
-    ALLOWED_HOSTS = ["localhost", "0.0.0.0", "127.0.0.1"]
+    ALLOWED_HOSTS = ["*", "localhost", "0.0.0.0", "127.0.0.1"]
 try:
     ALLOWED_CIDR_NETS = env.list("ALLOWED_CIDR_NETS")
     print(f"ALLOWED_CIDR_NETS: {ALLOWED_CIDR_NETS}")
```

Comment:
Use `ALLOWED_CIDR_NETS` with your subnets that the task can be launched in. Put them in Param store and pull down as environment variables.